uiowa/uiowa
# Force HTTPS locally and document local certificate install.

https://github.com/uiowa/uiowa/pull/2070 added an exception to the htaccess file to not redirect local.drupal.uiowa.edu requests. This is not ideal. It causes issues like loading assets over http (APR) and sessions (drush uli, logging people in/out as the site is navigated).

This PR removes that exception from htaccess and documents how to install an optional, local SSL certificate housed in the developer wiki. This is required for running Behat tests locally. 

### Testing
- Check out this branch locally.
- `composer install`
- `vagrant provision`
- Check that all http:// requests get redirected to https://.

Optional testing
- Download the certificates from the wiki: https://wiki.uiowa.edu/display/drupaldev/Local+Development+Environment+-+DrupalVM
- Follow the directions in the box/certs/README.md file.
- `vagrant provision`
- Hit some local URLs and see a real, live SSL cert.
- `blt behat`
